src/container/container_iv.c

```diff
@@ -456,8 +456,27 @@ cont_iv_ent_fetch(struct ds_iv_entry *entry, struct ds_iv_key *key,
 				D_ERROR("create cont prop iv entry failed "
 					""DF_RC"\n", DP_RC(rc));
 			} else if (class_id == IV_CONT_CAPA) {
-				/* Can not find the handle on leader */
-				rc = -DER_NONEXIST;
+				struct container_hdl	chdl;
+				int			rc1;
+
+				/* Try to find it through cont hdl cache */
```

Comment:
[Not requesting changes] Do I understand it correctly that here we have already looked up the cache and have not found anything?